docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -39,32 +38,23 @@ bigchaindb -y configure rethinkdb
 Show the values of the [BigchainDB node configuration settings](configuration.html).
 
 
-## bigchaindb export-my-pubkey
-
-Write the node's public key (i.e. one of its configuration values) to standard output (stdout).
-
-
 ## bigchaindb init
 
-Create a backend database (RethinkDB or MongoDB),
-all database tables/collections,
-various backend database indexes,
-and the genesis block.
+Create a backend database (Local MongoDB), all database tables/collections,
+various backend database indexes, and the genesis block.
 
 
 ## bigchaindb drop
 
-Drop (erase) the backend database (a RethinkDB or MongoDB database).
+Drop (erase) the backend database (a Local MongoDB database).
```

Comment:
Not sure if the document should include the word 'Local'. It was called `localmongodb` because the name `mongodb` was already there in the code. So, `localmongodb` is just an implementation detail. @ttmc ?

Reply:
I talked to @ttmc about this exactly, and he suggested that it should be localmongodb.

Reply:
What  I meant is that we should keep using `localmongodb` for now, because the different name helps us distinguish the old `mongodb` option from the new one (`localmongodb`). `localmongodb` is even the name of a subdirectory in the `bigchaindb/bigchaindb/backend/` directory. The different name makes it easy for us to distinguish the old from the new, for now. Maybe in the future, once all the old code is gone, we can shorten it to `mongodb`.

Anyway, in this line, we're not referring to that particular label, which is more of a code/configuration value. We're referring to a MongoDB database, the local one used by this node, so what you wrote is fine, but I'd write "Local" in all-lowercase and add a clarifying phrase:

`Drop (erase) the backend database (the local MongoDB database used by this node).`

(I also changed the article before `local` from `a` to `the` so it agrees with the article before  `backend database`.)